tests/test_wave_01.py

```diff
@@ -182,13 +188,14 @@ def test_moves_movie_from_watchlist_to_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
-    
-    raise Exception("Test needs to be completed.")
-    # *******************************************************************************************
-    # ****** Add assertions here to test that the correct movie was added to "watched" **********
-    # *******************************************************************************************
 
-@pytest.mark.skip()
+    # ADDED ASSERTIONS
+    assert updated_data["watchlist"] == [FANTASY_1]
+    assert updated_data["watched"] == [FANTASY_2, movie_to_watch]
```

Comment:
would it  be:
`assert FANTASY2 in updated_data[watched] and movie_to_watch in updated_data[watched]`

or
`assert for movie in [FANTASY_2, movie_to_watch] if movie in updated_data["watched"]`

Reply:
Yeah more like your first assertion, but I would break em up on two lines instead of using the `and` for readability.

```python
assert FANTASY_2 in updated_data["watched"] 
assert movie_to_watch] in updated_data["watched"]